src/Security/Member.php

```diff
@@ -871,6 +871,11 @@ public static function create_new_password()
      */
     public function onBeforeWrite()
     {
+        // Prevent spaces in emails
+        if($this->Email) {
```

Comment:
```suggestion
        if ($this->Email) {
```
Fixes PHPCS error https://travis-ci.com/github/silverstripe/silverstripe-framework/jobs/492253708